src/edit/items/renderPieceIdentifiers.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field } from 'redux-form';
+import { FormattedMessage } from 'react-intl';
+import _ from 'lodash';
```

Comment:
Would be better to import only noop
```suggestion
import { noop } from 'lodash';
```